test/parallel/test-child-process-spawnsync-timeout.js

```diff
@@ -28,7 +28,14 @@ const { debuglog, getSystemErrorName } = require('util');
 const debug = debuglog('test');
 
 const TIMER = 200;
-const SLEEP = common.platformTimeout(5000);
+let SLEEP = common.platformTimeout(5000);
+
+if (common.isWindows) {
+  // Some of the windows machines in the CI need more time to launch
+  // and receive output from child processes.
+  // https://github.com/nodejs/build/issues/3014
+  TIMEOUT = common.platformTimeout(15000);
```

Comment:
```suggestion
  SLEEP = common.platformTimeout(15000);
```

Reply:
oops, it seems I forgot to update the changes to the commit here, thanks for catching it